docs/modules/k6.md

```diff
@@ -40,6 +42,13 @@ func RunContainer(ctx context.Context, opts ...testcontainers.ContainerCustomize
 
 When starting the K6 container, you can pass options in a variadic way to configure it.
 
+#### Image
+
+If you need to set a different K6 Docker image, you can use `testcontainers.WithImage` with a valid Docker image
+for Cassandra. E.g. `testcontainers.WithImage("szkiba/k6x:v0.3.1")`.
```

Comment:
```suggestion
for k6x. E.g. `testcontainers.WithImage("szkiba/k6x:v0.3.1")`.
```

Reply:
Could it be like this instead?
```suggestion
for K6. E.g. `testcontainers.WithImage("szkiba/k6x:v0.3.1")`.
```

Ot making it explicit for k6x is better?

Reply:
Probably worth it to add it as an info/warning in the `Image` section?

Reply:
I would prefer to keep it simple and assume only `k6x` images are used. Introducing other variants will just make the usage of the module more complex.

Reply:
Ok, let's include a message in the docs regarding the usage of k6x.

I'm on PTO until Monday, could you elaborate a text that we can add to the docs explaining that? 🙏